supervised/automl.py

```diff
@@ -408,13 +127,16 @@ def load(self):
                     self._stacked_models += [models_map[stacked_model_name]]
 
             best_model_name = None
-            with open(os.path.join(self._results_path, "best_model.txt"), "r") as fin:
+            with open(os.path.join(path, "best_model.txt"), "r") as fin:
                 best_model_name = fin.read()
 
             self._best_model = models_map[best_model_name]
 
-            data_info_path = os.path.join(self._results_path, "data_info.json")
+            data_info_path = os.path.join(path, "data_info.json")
             self._data_info = json.load(open(data_info_path))
+            self.n_features = self._data_info["n_features"]
+            self.n_classes = self._data_info["n_classes"]
```

Comment:
`n_classes` will be confusing for regression tasks.

Reply:
Should I add a check to only add `n_classes` for classification tasks?

Reply:
I think yes, it is important to not bother the user with unnecessary things.

Reply:
Fixed this